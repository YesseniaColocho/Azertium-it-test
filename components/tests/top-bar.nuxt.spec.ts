import { describe, it, expect} from 'vitest'
import { mount } from '@vue/test-utils'

import topBar from '../top-bar.vue'

describe('top bar', () => {
    it('component renders top bar properly', () => {
      const wrapper = mount(topBar)
      expect(wrapper).toBeTruthy()
    })

    it('component html renders properly', () => {
      const wrapper = mount(topBar)
      expect(wrapper.html()).toMatchSnapshot()
    })
})