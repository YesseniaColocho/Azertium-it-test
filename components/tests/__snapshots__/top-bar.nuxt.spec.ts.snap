// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`top bar > component html renders properly 1`] = `"<section data-v-10c29057="" class="top-bar"><img data-v-10c29057="" class="image-azertium" src="/azertium-logo.png"></section>"`;
